server/sonar-web/src/main/js/apps/issues/sidebar/LanguageFacet.js

```diff
@@ -64,17 +66,17 @@ export default class LanguageFacet extends React.PureComponent {
     this.props.onChange({ [this.property]: [] });
   };
 
-  getLanguageName(language: string): string {
+  getLanguageName(language /*: string */) /*: string */ {
     const { referencedLanguages } = this.props;
     return referencedLanguages[language] ? referencedLanguages[language].name : language;
   }
 
-  getStat(language: string): ?number {
+  getStat(language /*: string */) /*: ?number */ {
     const { stats } = this.props;
     return stats ? stats[language] : null;
   }
 
-  handleSelect = (language: string) => {
+  handleSelect = (language /*: string */) => {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove parentheses around the parameter of this arrow function. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=javascript%3AS3524)